### ISSUE TITLE
lib工具库

### ISSUE THREAD

**Issue by Catcher-o** [Open-Debin/Emotion-FAN#41]
can I get it?

**Comment by Open-Debin** [comment]
Do you means this? https://github.com/Open-Debin/Emotion-FAN/tree/master/data/face_alignment_code/lib

**Comment by Catcher-o** [comment]
yes,thankyou!But I found that I would get stuck when running frame2face_afew.py. I want to know why. This is my screenshot.
![image](https://user-images.githubusercontent.com/55650085/187065917-5602e699-4045-4308-96b8-1a068c4a94c6.png)

**Comment by Momo-fg** [comment]
> yes,thankyou!But I found that I would get stuck when running frame2face_afew.py. I want to know why. This is my screenshot. ![image](https://user-images.githubusercontent.com/55650085/187065917-5602e699-4045-4308-96b8-1a068c4a94c6.png)

Hi, I also have this problem, did you finally solve it, I've been stuck many times. I would like to ask you how you solved it.
